depthai_nodes/ml/parsers/vehicle_attributes.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+import depthai as dai
+
+from ..messages.creators import create_multi_classification_message
+
+
+class MultiClassificationParser(dai.node.ThreadedHostNode):
+    """Postprocessing logic for Multiple Classification model.
+
+    Attributes
+    ----------
+    input : Node.Input
+        Node's input. It is a linking point to which the Neural Network's output is linked. It accepts the output of the Neural Network node.
+    out : Node.Output
+        Parser sends the processed network results to this output in a form of DepthAI message. It is a linking point from which the processed network results are retrieved.
+    classification_attributes : List[str]
+        List of attributes to be classified.
+    classification_labels : List[List[str]]
+        List of class labels for each attribute in `classification_attributes`
+
+    Output Message/s
+    ----------------
+    **Type**: CompositeMessage
+
+    **Description**: A CompositeMessage containing a dictionary of classification attributes as keys and their respective Classifications as values.
+    """
+
+    def __init__(
+        self,
+        classification_attributes: List[str],
+        classification_labels: List[List[str]],
+    ):
+        """Initializes the MultipleClassificationParser node."""
+        dai.node.ThreadedHostNode.__init__(self)
+        self.out = self.createOutput()
+        self.input = self.createInput()
+        self.classification_attributes: List[str] = classification_attributes
+        self.classification_labels: List[List[str]] = classification_labels
+
+    def run(self):
+        while self.isRunning():
+            try:
+                output: dai.NNData = self.input.get()
+            except dai.MessageQueue.QueueException:
+                break
+
+            layer_names = output.getAllLayerNames()
+
+            scores = []
+            for layer_name in layer_names:
+                scores.append(
+                    output.getTensor(layer_name, dequantize=True).flatten().tolist()
+                )
+
+            vehicle_attributes_message = create_multi_classification_message(
```

Comment:
Nitpick: Change variables names to not mention vehicles (also the one below this)